Sources/Dictionary/dictionaryManager.swift

```diff
@@ -0,0 +1,19 @@
+import OS
+
+public class DictionaryManager {
+	let gameWords = words
+
+	public init() {
+	}
+
+	public func getRandomIndex() -> Int {
```

Comment:
Read the whole error on that, it should tell you that that error is coming from a different package.